pkg/orm/subscriptionKey.go

```diff
@@ -41,6 +87,11 @@ func (a *SubscriptionKeyORM) GetSubscriptionKeysByMinerHotkey(hotkey string) ([]
 		return nil, err
 	}
 
+	// Cache the result
+	if err := cache.SetCache(subCache, apiKeys); err != nil {
+		log.Error().Err(err).Msgf("Error caching subscription keys")
+	}
+
 	return apiKeys, nil
```

Comment:
not sure if there is naming error within this function, but should apiKeys be renamed to subKeys?